pkg/cmd/render/render.go

```diff
@@ -631,3 +624,21 @@ func getInfrastructure(file string) (*configv1.Infrastructure, error) {
 	}
 	return config, nil
 }
+
+// If HA mode is None, set usingUnsafeScalingStrategy to scaling strategy.
+// Else if marker file exists set delayed bootstrap scaling strategy.
+func getNamespaceAnnotations(opts *renderOpts,infra *configv1.Infrastructure) map[string]string {
```

Comment:
we don't gain anything with this complexity I think we should only use the annotation to describe a non-standard install strategy.